src/main/java/io/curity/authenticator/netid/config/NetIdAccessConfig.java

```diff
@@ -45,7 +50,18 @@ public interface NetIdAccessConfig extends Configuration
     @Description("Sets the full path to the Net iD Access service.")
     String getPath();
 
-    HttpClient getHttpClient();
+    @Description("The Net iD Access service trust store.")
+    Optional<ServerTrustCryptoStore> getTrustStore();
```

Comment:
Does this mean that to configure the certificate to trust? Will a CA work?

Reply:
What do we use if empty?

Reply:
CA will work. If empty then cacerts store is used.

Reply:
Perhaps the description should include that. As well as the README